client/.eslintrc.js

```diff
@@ -1,42 +1,43 @@
 module.exports = {
-  "extends": ["airbnb", "prettier", "prettier/react"],
+  extends: ['airbnb', 'prettier', 'prettier/react'],
   // babel-eslint parser is used to support experimental features not supported in ESLint itself yet
-  "parser": "babel-eslint",
-  "parserOptions": {
-    "ecmaVersion": 2018,
-    "ecmaFeatures": {
-      "impliedStrict": true //enable global strict mode (if ecmaVersion is 5 or greater)
-    }
+  parser: 'babel-eslint',
+  parserOptions: {
+    ecmaVersion: 2018,
+    ecmaFeatures: {
+      impliedStrict: true, //enable global strict mode (if ecmaVersion is 5 or greater)
+    },
   },
-  "env": {
-    "browser": true,
-    "node": true,
-    "jest": true
+  env: {
+    browser: true,
+    node: true,
+    jest: true,
   },
-  "rules": {
+  rules: {
     //  allow .js extensions for JSX.
-    "react/jsx-filename-extension": [
+    'react/jsx-filename-extension': [
       1,
       {
-        "extensions": [".js", ".jsx"]
-      }
+        extensions: ['.js', '.jsx'],
+      },
     ],
-    "quotes": [
+    quotes: [
       2,
-      "single",
+      'single',
       {
-        "avoidEscape": true, // allows strings to use single-quotes or double-quotes so long as the string contains a quote that would have to be escaped otherwise
-        "allowTemplateLiterals": true // allows strings to use backticks
-      }
+        avoidEscape: true, // allows strings to use single-quotes or double-quotes so long as the string contains a quote that would have to be escaped otherwise
+        allowTemplateLiterals: true, // allows strings to use backticks
+      },
     ],
     // configure the prettier plugin
-    "prettier/prettier": [
-      "error",
+    'prettier/prettier': [
+      'error',
       {
-        "trailingComma": "es5",
-        "singleQuote": true
-      }
-    ]
+        trailingComma: 'es5',
+        jsxSingleQuote: true,
```

Comment:
@AymanAlqoqa 
I don't understand why did you change this file to use single quotes?
I prefer to keep the styling of the .eslintrc.js to use the syntax of the JSON of double quotation as the eslint config docs write js config files with it https://github.com/eslint/eslint/blob/master/.eslintrc.js
Also, where did `jsxSingleQoute` come from? I can't find it anywhere in the docs

Reply:
I used single qoute along all the file, is it ok to write jsx in double qoute while the rest of the file in single qoute??
jsxSingleQoute from documentaion of eslint.
Thanks

Reply:
@AymanAlqoqa  http://airbnb.io/javascript/react/#quotes

> jsxSingleQoute from documentaion of eslint. 
could you please provide me a link to it in the doc  `jsxSingleQoute`

Reply:
So, the best practice is to use double qoutes for jsx and single qoute for the rest of the file, i'll remove it as soon as possible.
Thanks alot :)

Reply:
@AymanAlqoqa it's not about the best practice, it's the about the style guide (airbnb) the config written for.